src/components/organisms/AppRouter/AdminSubrouter.tsx

```diff
@@ -74,7 +74,7 @@ export const AdminSubrouter: React.FC = () => {
         </Provided>
       </Route>
 
-      <Route path="/admin-ng">
+      <Route path={["/admin-ng", "/admin-ng/*"]}>
```

Comment:
NB: a catch all route for anything under `/admin-ng` that didn't get handled by previous routes.

Reply:
Removed in 2c33b6b8de126d7a4550fdd2ca5b20e1ad2b5168